### ISSUE TITLE
[TEST ISSUE] Where can I find the pytorch code that cause exceptions?

### ISSUE THREAD

**Issue by poweiw** [NVIDIA/TensorRT#4528]
I am tring to convert an AMP trained pytorch checkpoint to onnx and then tensorrt fp16, all process are ok, but when loading the tensorrt fp16 model for inference, it will generate Nan values.
I had use polygraphy to make test

 polygraphy run --model=onnx  mymodel.fp32.onnx.simplified  \    
    --trt --fp16  \    
    --trt-min-shapes x:[2,80,4] mask:[2,1,4] cond:[2.80,4]   \   
    --trt-opt-shapes x:[2,80,192] mask:[2,1,192] cond:[2,80,192]  \   
    --trt-max-shapes x:[2,80, 1000] mask:[2,1,1000] cond:[2,80,1000]  \   
    --save_outputs layer_outputs.json --trt-outputs mark all
the polygraphy report many errors.

[E] 9: Skipping tactic 0x0000000000 due to exception [shape.cpp:verify_output_type:1308] Mismatched type for tensor ONNXTRT_Broadcast_39_output', fp16 vs. expected type:fp32   
[E]10: Could not find any implementation for node {ForeignNode[/dit_blocks.15/attn/Cast_8...ONNXTRT_unsqueezeTensor_12228]}
[E] 10 [optimizer.cpp::computeCosts::4105] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[/dit_blocks.15/attn/Cast_8...ONNXTRT_unsqueezeTensor_12228]}.)
[!] Invalid Engine. Please ensure the engine was built correctly
my onnxruntime opset_version=18, and the exportion is :

torch.onnx.export(
       estimator, 
       (x, mask, cond),
       "my.onnx",
       export_params=True,
       opset_version=18,
       do_constant_folding=True,
       input_names = ['x', 'mask', 'cond', 't', 'spks'],
       output_names=['estimator_output'],
       dynamic_axes={'x': {2: 'seq_len'},  'mask':{2: 'seq_len'}, 'cond': {2: 'seq_len'}, 'estimator_output': {2: 'seq_len'}}
)
My question is , the polygraphy output tell that there are something wrong with my onnx exported model. but How do I convert the error message there to the right python code of model define or training pipeline?

Tensorrt V9.2 V10.0.1.6 are tested.
